### PR TITLE
Daily report for the trending maps

### LINKED ISSUE #5706: Add BCC support for the notify trending mail

### PR DESCRIPTION
**Pull request by ethervoid**
Closes #5706

### CODE REVIEW

File: app/mailers/reporter_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class ReporterMailer < ActionMailer::Base
+  default from: "cartodb.com <support@cartodb.com>"
```
Comment (Kartones) [inline app/mailers/reporter_mailer.rb]: Again.. this should be config based :/ I now takes time to do config changes but we shouldn't skip them... 
Even if OSS installs won't trigger the call

File: lib/cartodb/trending_maps.rb [hunk 2]
```diff
@@ -18,7 +18,8 @@ def get_trending_maps(simulation=false)
         yesterday_mapviews = stats_manager.get_api_calls_from_redis(username, {from: date, to: date, stat_tag: visualization_id})
         total_mapviews = stats_manager.get_total_api_calls_from_redis(username, visualization_id)
         if is_trending_map?(yesterday_mapviews[date_key], total_mapviews)
-          trending_maps[visualization_id] = { user: username, mapviews: total_mapviews }
+          visualization = Carto::Visualization.find(visualization_id)
+          trending_maps[visualization_id] = { user: username, user_mail: visualization.user.email, user_public_url: visualization.user.public_url, mapviews: total_mapviews, visualization_name: visualization.name }
```
Comment (Kartones) [inline lib/cartodb/trending_maps.rb]: multiline please, here Hound is correct, really hard to read in the diff :P

**Comment by Kartones** [comment]
Apart from the hardcoded email & one-liner big hash, +1

File: app/mailers/reporter_mailer.rb (hunk 1 above)
Comment (ethervoid) [inline app/mailers/reporter_mailer.rb]: Ok but that is this way in every mailer we have. The other mail I'm going to put in the config. I'll change all this too in a new property

File: lib/cartodb/trending_maps.rb (hunk 2 above)
Comment (ethervoid) [inline lib/cartodb/trending_maps.rb]: Ok

File: app/mailers/reporter_mailer.rb (hunk 1 above)
Comment (Kartones) [inline app/mailers/reporter_mailer.rb]: uhm, ok, but would be nice to have it via config anyway... but then as you see best

Comment (ethervoid) [inline app/mailers/reporter_mailer.rb]: Yup I'm changing and this way I learn how to add new configuration elements

**Comment by ethervoid** [comment]
Done the default from change for all the mailers and promoted the configuration to production. I'll deploy it when the groups thing is done.

**Pull request merged**